tests/tests/tier0/monitor-system-status/test_monitor_system_status.py

```diff
@@ -1,35 +1,66 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 import os
+import logging
 import time
-from typing import Dict
+from typing import Dict, List
 
 from bluechi_test.util import read_file
 from bluechi_test.test import BluechiTest
 from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
 from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
 
 
+LOGGER = logging.getLogger(__name__)
+
 node_one = "node-1"
 node_two = "node-2"
 node_three = "node-3"
 nodes = [node_one, node_two, node_three]
 
 
 def stop_all_agents(nodes: Dict[str, BluechiControllerContainer]):
+    LOGGER.info("Stopping all agents...")
     for node_name, node in nodes.items():
         result, output = node.exec_run("systemctl stop bluechi-agent")
         if result != 0:
             raise Exception(f"Failed to stop bluechi-agent on node '{node_name}': {output}")
 
 
 def start_all_agents(nodes: Dict[str, BluechiControllerContainer]):
+    LOGGER.info("Starting all agents...")
     for node_name, node in nodes.items():
         result, output = node.exec_run("systemctl start bluechi-agent")
         if result != 0:
             raise Exception(f"Failed to stop bluechi-agent on node '{node_name}': {output}")
 
 
+def check_events(ctrl: BluechiControllerContainer, expected_events: List[str]):
+    """
+    Continuously poll the current content in /tmp/events. As soon as the file
+    contains the same number of events as the expected_events, assert that the
+    content is the same - the order is important.
+    """
+
+    LOGGER.info("Checking events processed by the system monitor...")
+    while True:
```

Comment:
Processing timeout to break this never-ending loop?

Reply:
I added a small inline-comment to clarify this loop. 
If I'd define a custom timeout here and it triggers, the test has to be marked as failed - the same applies to the timeout from pytest. A custom timeout needs to be smaller than the one from pytest, so it might fail more often due to infrastructure issues in the CI, making it flaky.